packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -438,8 +438,21 @@ export default {
     },
     '$store.getters.selectedObject': {
       handler(selectedObject) {
-        if (selectedObject && !(selectedObject instanceof Event)) {
-          this.setView(VIEW_COMPONENT);
+        if (selectedObject) {
+          if (!(selectedObject instanceof Event)) {
+            this.setView(VIEW_COMPONENT);
+          }
+
+          if (
+            selectedObject instanceof Event &&
+            this.highlightedNodes.has(selectedObject.id)
+          ) {
+            this.currentTraceFilterIndex = Array.from(
```

Comment:
If the active selection is not a filtered event, can we hide this number from this display?

For example, this element:
![image](https://user-images.githubusercontent.com/8737782/149215824-2af6a97b-24a4-4645-8195-9001e7cd2832.png)


Should become:
![image](https://user-images.githubusercontent.com/8737782/149215869-34263f6f-107c-439c-a4a6-7852a6341f98.png)

If the user does not have a filtered event selected.

Reply:
fixed https://github.com/applandinc/appmap-js/pull/519/commits/c9bd5fee3eba5272ed597d26dd4775d7a8294dcc